concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3106,6 +3117,71 @@ jobs:
                   echo "Success: All VMs are up and running."
                 fi
 
+  - name: deploy-paas-audit-to-splunk
+    serial: true
+    plan:
+      - *add-grafana-job-annotation
+      - get: paas-audit-to-splunk
+        trigger: true
+
+      - in_parallel:
```

Comment:
Is there a reason that this `get: paas-audit-to-splunk` task isn't in the `in_parallel:`?

Reply:
Oversight :D Probably doesn't matter. But moved in for making it consistent.